entities/project.ts

```diff
@@ -76,6 +76,14 @@ class Project extends BaseEntity {
   @Column({ nullable: true })
   description?: string;
 
+  @Field({ nullable: true })
+  @Column({ nullable: true })
+  traceCampaignId?: string;
+
+  @Field(type => Boolean)
+  @Column({ default: false })
+  isTraceable: boolean;
```

Comment:
This field is redundant, the existence of `traceCampaignId` implies traceability of the project